configure

```diff
@@ -1384,8 +1384,8 @@ Optional Features:
                           information
   --enable-default-pie    build linux toolchain with default PIE
                           [--enable-default-pie]
-  --enable-multilib       build both RV32 and RV64 runtime libraries 
-                          [--disable-multilib]
+  --enable-multilib       build both RV32 and RV64 runtime libraries (only
+                          RV64 for musl libc) [--disable-multilib]
```

Comment:
This is caused by a forgotten change in `configure.ac`.
Can you update `configure.ac`?

Reply:
Yes, the new configure file is just generated by `$ autoconf configure.ac` :)